test/Integration/GetWork.ts

```diff
@@ -18,16 +18,16 @@ export class GetWork {
   @TestCase(TheRaven)
   @TestCase(TheMurdersInTheRueMorgue)
   @TestCase(AStudyInScarlet)
-  async getWork200(claim: Claim) {
-    const response = await this.client.getWork(claim.id)
+  async getWork200(signedVerifiableClaim: SignedVerifiableClaim) {
+    const response = await this.client.getWork(signedVerifiableClaim.id)
 
     Expect(response.status).toBe(200)
     Expect(response.ok).toBeTruthy()
 
     const body = await response.json()
 
-    Expect(body.id).toBe(claim.id)
-    Expect(body.attributes.name).toBe(claim.attributes.name)
+    Expect(body.id).toBe(signedVerifiableClaim.id)
+    Expect(body.attributes.name).toBe(Object(signedVerifiableClaim).claim.name)
```

Comment:
Why wrapping with `Object()`?

Reply:
Because claim is a typescript `object`, which does not allow you to retrieve anything inside the object itself.